tests/test_graphical_units/test_img_and_svg.py

```diff
@@ -41,4 +256,4 @@ def construct(self):
 
 @pytest.mark.parametrize("scene_to_test", get_scenes_to_test(__name__), indirect=False)
 def test_scene(scene_to_test, tmpdir, show_diff):
-    GraphicalUnitTester(scene_to_test[1], MODULE_NAME, tmpdir).test(show_diff=show_diff)
+    GraphicalUnitTester(scene_to_test[1], MODULE_NAME, tmpdir).test(show_diff=True)
```

Comment:
Why?

Reply:
Something I tried while getting the differences to show. I'll roll back that change at the end. Thanks for pointing it out.

Reply:
Great! Though in the future, I'd suggest not committing changes that are meant to be temporary. It makes review easier and this way you don't have to remember to roll it back :sweat_smile: